portfolio/src/main/webapp/script.js

```diff
@@ -28,7 +28,16 @@ function addRandomQuote() {
 
 async function getComments() {
   const response = await fetch('/data');
-  const commentsList = await response.text();
-  const commentsListElement = document.getElementById('comments-container')
-  commentsListElement.innerText = commentsList;
+  var commentsList = await response.text();
+
+  // Display comments in a list.
+  commentsList = JSON.parse(commentsList)
+  var commentsListElement = document.getElementById('comments-container')
+  var ul = document.createElement('ul');
+  for (var i = 0; i < commentsList.length; ++i) {
+    var li=document.createElement('li');
+    li.innerHTML = commentsList[i];
+    ul.appendChild(li);                                 
+  }
+  commentsListElement.appendChild(ul);
```

Comment:
might want to remove any other children that might exist before adding the updated one

Reply:
Hmm may I know why that's necessary? I initialised a new ul in the function, so shouldn't it start with an empty list each time?

Reply:
The li elements are fine, but seems like every time you run getComments() you'll be adding a new ul element to the 'comments-container', right?

Reply:
I see, I redid this part of the code to generate comment sections instead (inspired by Dalis' presentation :O). Please see the latest commit! The sentiment score has also been added in. Deployed at https://lguo-sps-summer20.df.r.appspot.com